sdk/monitor/monitor-ingestion/samples-dev/uploadCustomLogs.ts

```diff
@@ -28,10 +28,15 @@ export async function main() {
       AdditionalContext: "context",
     },
   ];
-  const result = await client.upload(ruleId, streamName, logs);
-  if (result.status !== "Success") {
-    console.log("Some logs have failed to complete ingestion. Upload status=", result.status);
-    for (const errors of result.errors) {
+  try {
```

Comment:
I think we may want to provide samples on how to use the error handling callback as well

Reply:
Yeah, we should show it both ways at least once

Reply:
Added samples